sources/packages/web/src/services/http/ApplicationChangeRequestApi.ts

```diff
@@ -14,4 +20,21 @@ export class ApplicationChangeRequestApi extends HttpBaseClient {
     const url = `application-change-request/${applicationId}`;
     await this.patchCall(this.addClientRoot(url), payload);
   }
+
+  /**
+   * Gets all pending application change requests (applications in 'Change pending approval' status).
+   * @param paginationOptions options to execute the pagination.
+   * @returns list of application change requests.
+   */
+  async getApplicationChangeRequests(
+    paginationOptions: PaginationOptions,
+  ): Promise<
+    PaginatedResultsAPIOutDTO<ApplicationChangeRequestPendingSummaryAPIOutDTO>
+  > {
+    let url = "application-change-request/pending?";
+    url += getPaginationQueryString(paginationOptions, true);
+    return this.getCall<
```

Comment:
Minor and not a blocker, the type is inferred and does not need to be explicitly added, IMO not adding it explicitly improves the readability.
![image](https://github.com/user-attachments/assets/237bf776-862f-4a8b-a182-f637b93b7f22)

@dheepak-aot FYI